Dockerfile

```diff
@@ -18,16 +18,22 @@ FROM golang:1.12.5 as builder
 ARG ARCH
 
 # Copy in the go src
-WORKDIR $GOPATH/src/sigs.k8s.io/cluster-api
+WORKDIR ${GOPATH}/src/sigs.k8s.io/cluster-api
 COPY pkg/    pkg/
 COPY cmd/    cmd/
 COPY vendor/ vendor/
 
+#create non-root user
+RUN groupadd -g 999 cluster-api-manager && \
```

Comment:
Can we make the username an `ARG`? It will be dryer... Cf. https://docs.docker.com/engine/reference/builder/#ARG

Reply:
@davidewatson Done!